test/automation/src/sql/notebook.ts

```diff
@@ -82,24 +98,24 @@ export class Notebook {
 	}
 
 	public async selectAllTextInRichTextEditor(): Promise<void> {
-		const editor = '.notebook-cell.active .notebook-preview[contenteditable="true"]';
+		const editor = `${activeCellSelector} .notebook-preview[contenteditable="true"]`;
 		await this.selectAllText(editor);
 	}
 
 	public async selectAllTextInEditor(): Promise<void> {
-		const editor = '.notebook-cell.active .monaco-editor';
+		const editor = `${activeCellSelector} .monaco-editor`;
 		await this.selectAllText(editor);
 	}
 
 	private async selectAllText(selector: string): Promise<void> {
 		await this.code.waitAndClick(selector);
-		await this.code.dispatchKeybinding(this.ctrlOrCmd + '+a');
+		await this.code.dispatchKeybinding(constants.ctrlOrCmd + '+a');
```

Comment:
Prefer we use interpolated strings for this kind of stuff

```
`${constants.ctrlOrCmd}+a`
```

It's just slightly easier to look at